js/webpack.config.babel.js

```diff
@@ -5,6 +5,7 @@ export default {
   output: {
     path: join(__dirname, '..', 'inst', 'www', 'shiny.react'),
     filename: 'shiny-react.js',
+    hashFunction: "sha256",
```

Comment:
Can you tell me more, why is this necessary? Ideally we'd fix your setup so it works without this line and perhaps update docs. If that's unpractical or we figure we should actually explicitly set `hashFunction`, please use single quotes (consistency) and add a comment (why such setting).

Reply:
From my understanding openssl removed support for the hashing algorithm _(md4)_ that older versions of webpack default to. In particular, with the one that's currently used by `shiny.react`.

So when running webpack it just fails.

![image](https://user-images.githubusercontent.com/211358/214328942-4dc15f1d-cab6-4b3b-a7c4-8a86732502fc.png)

(this is what happens when I run `yarn && yarn webpack` in master)

See [here](https://stackoverflow.com/a/69476335/302485)

If we update webpack to v5.54.0+ this problem goes away and we can remove this output option

**I just didn't want to update node packages along with this PR, but if you're ok with it I can.**

Reply:
I've updated the webpack and this solves the problem